Mathlib/Data/List/InsertIdx.lean

```diff
@@ -19,14 +19,72 @@ open Nat hiding one_pos
 
 namespace List
 
-universe u
+universe u v
 
-variable {α : Type u}
+variable {α : Type u} {β : Type v}
```

Comment:
Any reason why we can't drop the `universe` statement above and just do:
```suggestion
variable {α β : Type*}
```

Reply:
This file had `universe u .. Type u`, so I decided to follow the style. We may decide to change it everywhere later.